lib/iiif_print/engine.rb

```diff
@@ -1,6 +1,7 @@
 require 'active_fedora'
 require 'hyrax'
 require 'blacklight_iiif_search'
+require "bulkrax/version"
```

Comment:
I believe the issue is that bulkrax didn't get updated on this branch, so version doesn't exist at 3.5.1. 

I changed the test to say `if (!defined? Bulkrax::VERSION) || (Bulkrax::VERSION.to_i < 5)` and it works

Reply:
ok! thanks for chiming in. I'll close this PR then since it looks like yours is more flushed out. I'll admit I didn't spend too much time trying to understand it as well. 🙈